pkg/syncer/admissionhandler/validatepvcannotation.go

```diff
@@ -0,0 +1,87 @@
+package admissionhandler
+
+import (
+	"context"
+	"encoding/json"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/common"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
+)
+
+const (
+	CreatePVCWithAnnGuestClusterRequestedTopologyButEmptyValue = "Create PVC with AnnGuestClusterRequestedTopology " +
```

Comment:
can you print the actual annotation name in the message?

Reply:
Sure